cvat/apps/annotation/cvat_ddln.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2020
```

Comment:
Everything in this repository or folder is related to cvat, no need to repeat that verywhere, so `cvat_ddln` => `ddln`. But, the format implemented below is specific to Spotter aircraft annotation, for other tasks you will need a different format. So a better name may be `ddln_spotter.py` or similar.